### PR TITLE
renamed c1t -> cda1tenth

### PR DESCRIPTION
**Pull request by john-chrosniak**
# PR Details
## Description

This PR replaces references to CARMA1tenth (C1T) with CDA1tenth.

## Related GitHub Issue

NA

## Related Jira Key

[CF-947](https://usdot-carma.atlassian.net/browse/CF-947)

## Motivation and Context

CDA1tenth has been chosen as the name to encapsulate the Port Drayage proof-of-concept project and the follow-on workforce development project. So, we are replacing all references to the former CARMA1tenth (C1T) project to CDA1tenth.

## How Has This Been Tested?

Tested the port drayage demo on the vehicle with all renaming changes pulled to vehicle, infrastructure, and Raspberry Pis.

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

[CF-947]: https://usdot-carma.atlassian.net/browse/CF-947?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,34 +16,34 @@ The following configuration steps must be completed before a C1T vehicle or simu
...
-- `record_bag` is a boolean argument for recording all data from the system in a ROS2 rosbag and defaults to `false`. Bags are saved in `~/c1t_bags` along with a copy of the parameter file used
+- `record_bag` is a boolean argument for recording all data from the system in a ROS2 rosbag and defaults to `false`. Bags are saved in `~/cda_bags` along with a copy of the parameter file used
 
 
 ## Post launch
 
-After launching the nodes included in `c1t_bringup`, additional actions are needed to finalize the system on a physical vehicle.
+After launching the nodes included in `cda1tenth_bringup`, additional actions are needed to finalize the system on a physical vehicle.
 
 1. Send an empty message to the `/ackermann_cmd` topic to initialize the VESC by publishing via command line on the vehicle with `ros2 pub /ackermann_cmd ackermann_msgs/msg/AckermannDriveStamped`. Use `CTRL + C` after a few have sent and the original terminal output has updated.
```
Comment (willjohnsonk) [inline README.md]: Should we remove this? Or put it as a footnote for reference

**Review by willjohnsonk — COMMENTED** [review]
Minor comment

File: README.md (hunk 1 above)
Comment (john-chrosniak) [reply README.md]: I think we can remove this and include it in a debugging document of some sorts

**Review by willjohnsonk — APPROVED** [review]
tested on vehicle

**Pull request merged**